test/import/FSHTank.test.ts

```diff
@@ -527,6 +772,19 @@ describe('FSHTank', () => {
       ).toBeUndefined();
     });
 
+    it('should find an invariant when fishing with a version', () => {
+      expect(tank.fish('Invariant1|1.0.0', Type.Invariant).name).toBe('Invariant1'); // tank.config.version = 1.0.0
+    });
+
+    // TODO - add back in once rules on invariants are supported
+    it.skip('should find an invariant when fishing with a version that is set by a caret rule', () => {
+      expect(tank.fish('Invariant2|v-2.0.0', Type.Invariant).name).toBe('Invariant2');
+    });
+
+    it('should not find an invariant when fishing with a version that does not match', () => {
+      expect(tank.fish('Invariant1|2.0.0', Type.Invariant)).toBeUndefined();
+    });
+
```

Comment:
I think these can be removed. Also, Invariants don't support caret rules anyway. :-)

Reply:
Part of [0d0852d](https://github.com/FHIR/sushi/pull/1275/commits/0d0852d19cbf5a24e3e63fdf9a7f359ac288bcce)